src/exec.c

```diff
@@ -61,7 +61,9 @@
 #include <sys/poll.h>
 
 #ifdef __CYGWIN__
-    #define NOGDI
+    /* #define NOGDI */
```

Comment:
If the define is not longer needed, I would rather it was removed than commented out.  I guess someone should check the behaviour of a handful of CYGWIN versions .